demo-shell/src/app/components/app-layout/app-layout.component.html

```diff
@@ -25,6 +25,8 @@
                     <adf-language-menu></adf-language-menu>
                 </mat-menu>
 
+                <adf-notification-history [menuPositionX]="'before'" [menuPositionY]="'above'"></adf-notification-history>
```

Comment:
If the value is not an expression but just a string, you can simplify the binding
```suggestion
                <adf-notification-history menuPositionX="before" menuPositionY="above"></adf-notification-history>
```